RecoHI/HiEvtPlaneAlgos/src/HiEvtPlaneFlatProducer.cc

```diff
@@ -98,12 +130,18 @@ HiEvtPlaneFlatProducer::HiEvtPlaneFlatProducer(const edm::ParameterSet& iConfig)
       trackTag_(iConfig.getParameter<edm::InputTag>("trackTag")),
       FlatOrder_(iConfig.getParameter<int>("FlatOrder")),
       NumFlatBins_(iConfig.getParameter<int>("NumFlatBins")),
+      minvtx_(iConfig.getParameter<double>("minvtx")),
+      maxvtx_(iConfig.getParameter<double>("maxvtx")),
+      flatnvtxbins_(iConfig.getParameter<int>("flatnvtxbins")),
+      flatminvtx_(iConfig.getParameter<double>("flatminvtx")),
+      flatdelvtx_(iConfig.getParameter<double>("flatdelvtx")),
       caloCentRef_(iConfig.getParameter<double>("caloCentRef")),
       caloCentRefWidth_(iConfig.getParameter<double>("caloCentRefWidth")),
       CentBinCompression_(iConfig.getParameter<int>("CentBinCompression")),
-      Noffmin_(iConfig.getParameter<int>("Noffmin")),
-      Noffmax_(iConfig.getParameter<int>("Noffmax")),
       useOffsetPsi_(iConfig.getParameter<bool>("useOffsetPsi")) {
+  //  UseEtHF = kFALSE;
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant